code/extensions/CommentsExtension.php

```diff
@@ -423,7 +423,18 @@ public function CommentsForm() {
 			Requirements::javascript(THIRDPARTY_DIR . '/jquery-validate/lib/jquery.form.js');
 			Requirements::javascript(COMMENTS_THIRDPARTY . '/jquery-validate/jquery.validate.min.js');
 			Requirements::javascript('comments/javascript/CommentsInterface.js');
```

Comment:
I realised when adding the Timeago library that the number of JS files was starting to get unwieldy.  Which files do you propose concatenating?  The jQuery libraries may of course be used by other modules.  CommentsInterface.js and jquery.timeago.js perform different functions.  I guess comments.timeago.js could be merged with CommentsInterface.js, it simply primes the Timeago functionality when a document is ready.

Oh or are you referring to all the language files?